cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -385,15 +388,29 @@ func (s *CMServer) Run(_ []string) error {
 		}
 	}
 
+	volumePlugins := ProbeRecyclableVolumePlugins(s.VolumeConfigFlags)
+	provisioner, provisionerExpected, err := NewVolumeProvisioner(volumePlugins, cloud, s.VolumeConfigFlags)
+	if provisionerExpected && err != nil {
+		glog.Fatalf(fmt.Sprint("%v", err))
```

Comment:
I'll add some more context.  The error itself contains all the pertinent stuff, but I can see how this code looks ... wrong.

Can't just Fatalf(err), though.  It's not a string.